source/SpinalHDL/Structuring/parametrization.rst

```diff
@@ -21,17 +21,19 @@ Here is an example of class parameters
 
 .. code-block:: scala
 
-  case class MyBus(width : Int) extends Bundle{
+  case class MyBus(width : Int) extends Bundle {
     val mySignal = UInt(width bits)
   }  
   
 .. code-block:: scala
 
-  case class MyComponent(width : Int) extends Component{
+  case class MyComponent(width : Int) extends Component {
     val bus = MyBus(width)
   }
   
-You can also use global variable defined in scala object's, but note that recently was added the ScopeProperty feature which improve on that solution.
+You can also use global variables defined in scala object's (which make them static like a
+`companion object` pattern), but note that recently was added the ScopeProperty feature
```

Comment:
```suggestion
`companion object`). A ``ScopeProperty`` can also be used for configuration.
```
?

Reply:
I am going to move this change to the other commit/issue #190 as I have tried to improve it here and over there with different results :) 

Yeah I guess someone added it so there is a reference to it somewhere in the docs.

Maybe it needs a sentence to explain how ScopeProperty is better and why with an example if possible or a pointer/link to it in action in the code-base.  I do not know anything about this feature at this time to be able to improve that situation.

Maybe keep the reference to it existing in the docs here and open an issue calling for clarification around this feature.